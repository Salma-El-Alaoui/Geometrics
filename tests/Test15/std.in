LOAD test15-1.txt
LIST
CLEAR
LIST
LOAD test15-2.txt
LOAD test15-3.txt
EXIT

