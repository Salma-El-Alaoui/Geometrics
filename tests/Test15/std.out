OK
#Fichier charge
C c1 1 1 1
L l1 1 1 1 1
OA oa1 c1 l1 r1
PL pl1 1 1 1 1
R r1 1 1 1 1
OK
#Modele cleared
ERR
#Le fichier contient au moins un objet agrege invalide.
ERR
#Le fichier contient au moins une commande ayant un nombre de parametres invalide.

